frontend/src/components/store/PluginCard.tsx

```diff
@@ -32,7 +32,6 @@ const PluginCard: FC<PluginCardProps> = ({ storePlugin, installedPlugin }) => {
 
   return (
     <div
-      className="deckyStoreCard"
```

Comment:
These classes seemed unused.

Reply:
they exist for theme developers, keep them

Reply:
@AAGaming00 done. Also added a small warning documenting this.